test/subspace-test-service/src/lib.rs

```diff
@@ -248,7 +248,7 @@ pub struct MockConsensusNode {
     next_slot: u64,
     /// The slot notification subscribers
     #[allow(clippy::type_complexity)]
```

Comment:
Maybe was a complex type before, but looks normal to me now, probably suppression can be removed.